docs/gettingstarted/debugprofiler.md

```diff
@@ -0,0 +1,62 @@
+# Debug Profiler
+
+Minecraft provides a Debug Profiler that can be used to find time consuming code. Specially considering things like `TickEvents` and Ticking `TileEntities` this can be very usefull for modders and server owners that want to find a lag source.
+
+## Using the Debug Profiler
+
+The Debug Profiler is very simple to use, it requires two commands `/debug start` and `/debug stop`
+They are booth pretty self explaining, but you run `/debug start` to start the profiling process and then run `/debug stop` to stop it again.
+The important part here is that the more time you give it to collect the data the better the results will be.
+It is recommended to at least let it collect data for a minute.
+
+!!! note
+  In case you want to test your `TileEntities` you need to place them into the world.
+
+After you've stopped the debugger it will create a new file, it can be found within a new directory called `debug` in your run directory.
+The name will look like `profile-results-YEAR-MONTH-DAY_HOUR.MINUTE.SECOND` it's a simple `.txt`
```

Comment:
`new directory` implies that the user has not used the debugger before. I would suggest:

When the debugger is stopped, a text file containing the profiling result will be created in the `debug` subdirectory of the run directory.
The file name will be formatted with the date and time as `profile-results-yyyy-mm-dd_hh.mi.ss.txt`.